ts/features/bonus/bpd/api/patchedTypes.ts

```diff
@@ -24,3 +24,21 @@ export const PatchedCitizenResource = t.intersection(
 );
 
 export type PatchedCitizenResource = t.TypeOf<typeof PatchedCitizenResource>;
+
+/**
+ * patched version of CitizenV2Resource
+ * - technicalAccount must be optional
+ */
+// required attributes
+const PatchedCitizenV2ResourceO = t.interface({
+  technicalAccount: t.string
+});
+
+export const PatchedCitizenV2Resource = t.intersection(
+  [PatchedCitizenResourceR, PatchedCitizenResourceO, PatchedCitizenV2ResourceO],
+  "PatchedCitizenResource"
```

Comment:
```suggestion
  "PatchedCitizenResourceV2"
```
It could be useful for debug purposes

Reply:
Thank you.
I don't know how this can be used, and what is it's purpose. Can you give me some hint?
Modified in [e2fae62](https://github.com/pagopa/io-app/pull/2969/commits/e2fae6228449c931e1f099babeca1b5ad7936215)

Reply:
Of course! It used in error reporting, here https://github.com/gcanti/io-ts/blob/cc9b6edae8722b31e97c2f22e5c31cc8131eeadd/src/PathReporter.ts#L18 for example
I guess it is used also as identifier (I'm not so sure)